examples/cgroup_skb/main.go

```diff
@@ -0,0 +1,130 @@
+//go:build linux
+// +build linux
+
+// This program demonstrates attaching an eBPF program to a control group.
+// The eBPF program will be attached as an egress filter for the socket to count the packets.
+// It prints the count of total packets every second.
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/link"
+	"github.com/cilium/ebpf/rlimit"
+)
+
+// $BPF_CLANG and $BPF_CFLAGS are set by the Makefile.
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc $BPF_CLANG -cflags $BPF_CFLAGS bpf ./bpf/cgroup_skb_example.c -- -I../headers
+
+const (
+	mapKey      uint32 = 0
+	procFS             = "/proc/mounts"
+	bpfProgName        = "count_egress_packets"
+)
+
+var cgroup2Mount = struct {
```

Comment:
For the example its OK to skip the `sync.Once` and memoizing the result of the lookup I think: less code in an example is better.